### PR TITLE
Uml 840: Minor dac changes applied

### PR DESCRIPTION
**Pull request by SeemaMenon**
# Purpose

https://opgtransform.atlassian.net/browse/UML-840

Fixes UML-840

## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/app/src/Actor/templates/actor/check-access-codes.html.twig [hunk 1]
```diff
@@ -63,7 +63,7 @@
 
                         {% if containsActive %}
 
-                            <h2 class="govuk-heading-m">{% trans %}Active codes{% endtrans %}</h2>
+                            <h3 class="govuk-heading-m">{% trans %}Active codes{% endtrans %}</h3>
```
Comment (GemTay) [inline service-front/app/src/Actor/templates/actor/check-access-codes.html.twig:66]: "Active codes" an "inactive codes" should be a h2 as the ticket states, the govuk-accordion__section-heading is already a h3 (line 3 in check-code-details.html.twig) so the ticket is no longer needed

**Review by GemTay — CHANGES_REQUESTED** [review]
H3's just need changing back to h2's

File: service-front/app/src/Actor/templates/actor/check-access-codes.html.twig (hunk 1 above)
Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/check-access-codes.html.twig]: I have no clue what my understanding was...corrected back.

**Review by SeemaMenon — COMMENTED** [review]
@GemTay - reverted changes. My bad !

**Review by GemTay — APPROVED** [review]

**Pull request merged**